sway/config.c

```diff
@@ -243,6 +243,7 @@ static void config_defaults(struct sway_config *config) {
 	config->default_layout = L_NONE;
 	config->default_orientation = L_NONE;
 	if (!(config->font = strdup("monospace 10"))) goto cleanup;
+	config->font_description = pango_font_description_from_string(config->font);
```

Comment:
Can we call the `font` command here with arg `"monospace 10"`?

We'd be able to de-duplicate some font-configuration bits, and also drop the
extra call to `config_update_font_height` below (which is kind of a hack). In
fact, we can move the logic from `config_update_font_height` into the font
command itself, since the font height only changes when the font changes, and
drop the separate `config_update_font_height` entirely.

Does this seem reasonable?

Reply:
I'll re-think how to optimise that further and address it separately then. It's a non-issue in this scope.